object_detection/ks-app/environments/base.libsonnet

```diff
@@ -0,0 +1,4 @@
+local components = std.extVar("__ksonnet/components");
+components + {
+  // Insert user-specified overrides here.
```

Comment:
Instead of only adding comments here, how about adding an example here?

like 
```
// Insert user-specified overrides here.
// user override data???
```

Reply:
actually the comment is automatically generated by "ks init"

Reply:
Ah, I see, thanks.